app/src/main/res/values-af/strings.xml

```diff
@@ -10,8 +10,7 @@
   <string name="menu_fullscreen">Volle skerm</string>
```

Comment:
Strings are auto generated so don't commit these changes.

Reply:
Ok. I will not commit these changes. Can you tell me where this app can be translated?